cli/command/image/history.go

```diff
@@ -45,12 +48,28 @@ func NewHistoryCommand(dockerCli command.Cli) *cobra.Command {
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only show image IDs")
 	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Don't truncate output")
 	flags.StringVar(&opts.format, "format", "", flagsHelper.FormatHelp)
+	flags.StringVar(&opts.platform, "platform", "",
+		`Specify a platform from a multi-platform image to show the history for.
+If the platform is not specified, the host platform is preferred if it's available, otherwise any available platform is used.
```

Comment:
@dvdksn I applied your suggestions; I'm slightly wondering if we should / could condense the information a bit; for example;

`Specify a platform` -> `Platform from a multi-platform image ...`

(I don't think we generally add `Specify ....` as that's a bit implicit for options?)


The "format" could possibly be included in the first line, something like;

Platform from a multi-platform image to show the history for (format: "os[/arch[/variant]]")

By default, the daemon's native platform is preferred, otherwise any available platform is used.
```

(wondering if `the host platform` is ambiguous, and if we need to emphasise `daemon` platform)

Reply:
i also think we should probably change "any available platform is used" to something more precise.. what does "any" mean in this context? the first available platform in alphabetical order?

Reply:
@thaJeztah your suggestion sgtm. Coming back to this I realize that multi-line flag descriptions are kinda exotic - we don't use them a lot. Maybe we should instead create sections in the respective `.md` files for the `platform` flag, like we usually do, to show examples.

Reply:
IMO, if there is a specific behavior we've decide is part of the contract "i.e. it's always the first alphabetical one", then we should make that explicit. On the other hand, if we explicitly don't want to make any future guarantees about **which platform** is used, then **any** is fine. I honestly kind of would rather just return an error if there's no platform for the given image, but 🤷‍♀️ it's not a big deal either way, especially for `history`.